sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Constructors.scala

```diff
@@ -94,6 +94,37 @@ case class ST_PolygonFromText(inputExpressions: Seq[Expression])
   }
 }
 
+/**
+  * Return a line from a string. The string must be plain string and each coordinate must be separated by a delimiter.
+  *
+  * @param inputExpressions
+  */
+case class ST_LineFromText(inputExpressions: Seq[Expression])
```

Comment:
This implementation is not consistent with PostGIS: https://postgis.net/docs/ST_LineFromText.html

The input string must be WKT. The logic should simply re-use the ST_GeomFromWKT

Reply:
I referred ST_GeomFromWKT  function call and added a LINE filter. Let me know if it needs modification or refactoring.